msctl

```diff
@@ -1209,9 +1220,15 @@ start() {
     exit 1
   fi
   # Start the server.
-  sh -c "tail -f --pid=\$$ '$WORLD_DIR/console.in' | {
-    $SERVER_COMMAND mscs-world='$1' > '$WORLD_DIR/console.out' 2>&1; kill \$$;
-    }" &
+  if [ $RUN_IN_FOREGROUND -eq 1 ]; then
+    sh -c "tail -f --pid=\$$ '$WORLD_DIR/console.in' | {
+      $SERVER_COMMAND mscs-world='$1' > '$WORLD_DIR/console.out' 2>&1; kill \$$;
+      }"
+  else
+    sh -c "tail -f --pid=\$$ '$WORLD_DIR/console.in' | {
+      $SERVER_COMMAND mscs-world='$1' > '$WORLD_DIR/console.out' 2>&1; kill \$$;
+      }" &
+  fi
   # Verify the server is running.
```

Comment:
Great catch, I didn't think of that. Do you like to have a new pull-request with more or less only the dockerfiles? If you like I can also a docker-readme file with more examples.

Just for clarification when running inside docker container you would normally only run one world per container. So running multiple worlds inside one container wouldn't make that much seans. To run multiple worlds you would do something like:

    docker build -t mscs .
 
    <create the worlds volumes, init and setup the config for the worlds>

    docker run -d -t -p 25565:25565 -v world1:/opt/mscs --name=world1 mscs
    docker run -d -t -p 25566:25565 -v world2:/opt/mscs --name=world2 mscs